stdlib/2and3/operator.pyi

```diff
@@ -34,8 +34,8 @@ def is_(a: Any, b: Any) -> bool: ...
 
 def is_not(a: Any, b: Any) -> bool: ...
 
-def abs(x: SupportsAbs) -> Any: ...
-def __abs__(a: SupportsAbs) -> Any: ...
+def abs(x: SupportsAbs[Any]) -> Any: ...
+def __abs__(a: SupportsAbs[Any]) -> Any: ...
```

Comment:
IIUC these can be turned into `SupportsAbs[_T] -> _T`

Reply:
Good point, done.